README.rst

```diff
@@ -34,6 +34,61 @@ One Time Setup
   # Here's how you might do that if you have virtualenvwrapper setup.
   mkvirtualenv -p python3.8 learning-assistant
 
+In your ``private.txt`` requirements file in edx-platform, add:
```

Comment:
I think it would be helpful to enumerate which services this plugin depends on - the LMS and discovery. Both you and Marcos ran into issues with the discovery container not running/working, which led to issues reading from the cache. If we instruct users to have both running, that can help avoid issues.